src/main/java/ca/uhn/fhir/jpa/starter/service/NotificationDataSource.java

```diff
@@ -522,7 +522,42 @@ public List<OrgIndicatorAverageResult> getOrgIndicatorAverageResult(List<String>
 
 	}
 
-	public Double getCacheValueSumByDateRangeIndicatorAndMultipleOrgId(Date from, Date to, String indicator,
+	public List<OrgIndicatorAverageResult> getCacheValueAverageWithoutZeroByDateRangeIndicatorAndMultipleOrgIdScorecard(List<String> orgIds, List<String> indicators, Date startDate, Date endDate) {
```

Comment:
I see this function not being used, see if there is any typo.

Reply:
It gets called by Reflection